backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/project/budget/BudgetProjectRestController.java

```diff
@@ -61,7 +61,7 @@ public BudgetProjectRestController(@NonNull final BudgetProjectRestService proje
 			@ApiResponse(code = 403, message = "Accessing the resource you were trying to reach is forbidden"),
 			@ApiResponse(code = 422, message = "The request entity could not be processed")
 	})
-	@PostMapping
+	@PutMapping
```

Comment:
what's the reason for switching from POST to PUT?
(asking for curiosity)

@metas-ts  do we have a guideline/convention for that?

Reply:
well, there was a discussion at some point and since this is also doing an update we concluded that it should be a `PUT` -> also we are already using `PUT` for upserting in other controllers, e.g. BPartnerRestController